src/MessageList/MessageListBox.vala

```diff
@@ -66,4 +66,8 @@ public class Mail.MessageListBox : Gtk.ListBox {
             current_node = (Camel.FolderThreadNode?) current_node.child;
         }
     }
+
+    public void reply () {
+        prepend (new InlineComposer ());
```

Comment:
shouldn't this be an add and then we scroll to bottom since new messages will appear below old messages?